File: app/src/test/java/de/tum/in/tumcampusapp/activities/CurriculaActivityTest.java
package de.tum.in.tumcampusapp.activities;

import org.junit.runner.RunWith;
import org.robolectric.RobolectricTestRunner;
import org.robolectric.annotation.Config;

import de.tum.in.tumcampusapp.BuildConfig;
import de.tum.in.tumcampusapp.TestApp;

@RunWith(RobolectricTestRunner.class)
@Config(constants = BuildConfig.class, application = TestApp.class)
public class CurriculaActivityTest extends BaseActivityTest {

    @Override
    public void mainComponentDisplayedTest() {
        // idIsDisplayed(R.id.activity_curricula_list_view);
    }
}

File: app/src/test/java/de/tum/in/tumcampusapp/activities/CurriculaActivityTest.kt
package de.tum.`in`.tumcampusapp.activities

import de.tum.`in`.tumcampusapp.BuildConfig
import de.tum.`in`.tumcampusapp.TestApp
import org.junit.runner.RunWith
import org.robolectric.RobolectricTestRunner
import org.robolectric.annotation.Config

@RunWith(RobolectricTestRunner::class)
@Config(constants = BuildConfig::class, application = TestApp::class)
class CurriculaActivityTest : BaseActivityTest() {

    override fun mainComponentDisplayedTest() {
        // idIsDisplayed(R.id.activity_curricula_list_view);
    }
}
